.github/workflows/main.yml

```
@@ -132,14 +132,14 @@ jobs:
     - run: git fetch --prune --unshallow
     - name: Set up Python ${{ matrix.python }}
       uses: actions/setup-python@v2
+      if: matrix.python != '3.9-dev'
+      with:
+        python-version: ${{ matrix.python }}
+    - name: install python ${{ matrix.python }}
```

Great to see an action for this and remove a bunch of config!

---

Perhaps keep this consistent with the non-3.9 name:
```suggestion
    - name: Set up Python ${{ matrix.python }}
```